docs/UserGuide.adoc

```diff
@@ -243,6 +243,30 @@ Format: `clearhistory`
 After the command history is cleared, you will not be able to undo any previous commands.
 ====
 
+=== Adding remarks on a person: `remark`
```

Comment:
"...**to** a person" --> do people usually add remarks "on" other people? I guess you got the idea from something like "He remarked on the fact that I was being anal" but in this case I think "to" is more appropriate